client/app/pages/queries-list/queries-list.html

```diff
@@ -1,6 +1,10 @@
 <div class="container">
-  <page-header title="'Queries'"></page-header>
-
+  <div ng-switch="$ctrl.currentPage">
+    <div ng-switch-when="my"><page-header title="My queries"></page-header></div>
```

Comment:
`PageHeader` component is now in React, and `title` should be an expression (limitation of `react2angular`):
```html
<page-header title="'My queries'">
```
Currently there are related errors in browser console:
![image](https://user-images.githubusercontent.com/12139186/51759898-f5774980-20d1-11e9-838a-0b9474e3cb9d.png)

Reply:
And probably instead of using `ng-switch` it's better to add title to route's `resolve` object:
```javascript
...
resolve: {
  currentPage: () => 'archive',
  title: 'Archived queries',
...
```
and then just use it
```html
<page-header title="$ctrl.title">
```

Reply:
@kravets-levko Moving the title into the resolve object gives me:

```
Warning: Failed prop type: The prop `title` is marked as required in `PageHeader`, but its value is `undefined`.
```

I'm a bit confused how this can be, and how this relates to the already existing `title` value that is returned by the `init` function return objects, e.g. https://github.com/getredash/redash/blob/ff42ec2cc6700023e48c85fde387a955700b507a/client/app/pages/queries-list/index.js#L57?

Reply:
@jezdez I think resolve expects a function. But you can also map the `currentPage` value to a title in the controller itself and set a value on the controller, which you will use in the template.

Reply:
@jezdez Add a `title` attribute to routes similar as in queries list:
https://github.com/getredash/redash/blob/ff42ec2cc6700023e48c85fde387a955700b507a/client/app/pages/queries-list/index.js#L57
Then inject `$route` to controller and use current route's title:
```javascript
this.title = $route.current.title; // will make it available as $ctrl.title
```
This will also properly update document title.

Reply:
@kravets-levko Nice, that helped refactoring this a bunch! See my latest commit.